integrations/docker/images/chip-build-bouffalolab/setup.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -x
+CURRENT_DIR=$(
+    cd "$(dirname "$0")"
+    pwd
+)
+SDK_ROOT_DIR=$CURRENT_DIR/../
+
+TOOLCHAIN_SETUP_ROOT=~/.bouffalolab/toolchain
```

Comment:
$HOME is not ideal. use something like `/opt/bouffallolab-sdk` or similar. The SDK is likely shared among all users of a system.

Reply:
Updated.